tests/installation/disable_boot_menu_timeout.pm

```diff
@@ -0,0 +1,18 @@
+# SUSE's openQA tests
```

Comment:
(optional) could we create intermediate folder under /installation/ to put there all test related with bootloader settings?